### PR TITLE
Update/passive rg

### PR DESCRIPTION
**Pull request by LeStarch**
| | |
|:---|:---|
|**_Originating Project/Creator_**| |
|**_Affected Component_**|  |
|**_Affected Architectures(s)_**|  |
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**|  |
|**_Builds Without Errors (y/n)_**|  |
|**_Unit Tests Pass (y/n)_**|  |
|**_Documentation Included (y/n)_**|  |

---
## Change Description

Adds a passive rate group!

### CODE REVIEW

**Comment by ThibFrgsGmz** [comment]
Wouldn't it be better to have a sequence diagram in mermaid?

File: Svc/PassiveRateGroup/docs/PassiveRateGroup.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<title>PassiveRateGroup Component Dictionary</title>
+# PassiveRateGroup Component Dictionary
```
Comment (timcanham) [inline Svc/PassiveRateGroup/docs/PassiveRateGroup.md:2]: These are artifacts of the old make system that were autogenerated. Were you planning to add the autogeneration back in?

File: Svc/PassiveRateGroup/docs/sdd.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<title>PassiveRateGroup Component SDD</title>
+# RateGroupDriver Component
+
+## 1. Introduction
+
+`Svc::PassiveRateGroup` is an passive component that drives a set of components connected to `Svc::Sched` output ports. It contains an synchronous input `Svc::Cycle` port that drives all the operations.  The component invokes each output port in order, passing an argument indicating the order. It tracks execution time of the cycle.
```
Comment (timcanham) [inline Svc/PassiveRateGroup/docs/sdd.md:6]: Maybe this was originally my poor writing, but change the second to last sentence to say "passing an argument specified in the supplied context list."

File: Svc/PassiveRateGroup/test/mod.mk [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#
```
Comment (timcanham) [inline Svc/PassiveRateGroup/test/mod.mk:1]: delete?

File: Svc/PassiveRateGroup/test/ut/PassiveRateGroupImplTester.cpp [hunk 4]
```diff
@@ -0,0 +1,94 @@
+/*
```
Comment (timcanham) [inline Svc/PassiveRateGroup/test/ut/PassiveRateGroupImplTester.cpp:1]: Remove ITAR headers.

File: Svc/PassiveRateGroup/test/ut/Readme.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+This test can be run by executing the following:
```
Comment (timcanham) [inline Svc/PassiveRateGroup/test/ut/Readme.txt:1]: Remove file

File: Svc/PassiveRateGroup/test/ut/mod.mk [hunk 6]
```diff
@@ -0,0 +1,35 @@
+#
```
Comment (timcanham) [inline Svc/PassiveRateGroup/test/ut/mod.mk:1]: Remove

File: Svc/PassiveRateGroup/test/ut/runtest_CYGWIN [hunk 7]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```
Comment (timcanham) [inline Svc/PassiveRateGroup/test/ut/runtest_CYGWIN:1]: remove

File: Svc/PassiveRateGroup/test/ut/runtest_DARWIN [hunk 8]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```
Comment (timcanham) [inline Svc/PassiveRateGroup/test/ut/runtest_DARWIN:1]: remove

File: Svc/PassiveRateGroup/test/ut/runtest_LINUX [hunk 9]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```
Comment (timcanham) [inline Svc/PassiveRateGroup/test/ut/runtest_LINUX:1]: remove

**Review by timcanham — CHANGES_REQUESTED** [review]
Leftover files, old headers and other minor comments.

**Comment by LeStarch** [comment]
@timcanham  ready for re-review.  Your comments and @ThibFrgsGmz's comments addressed.

File: Svc/PassiveRateGroup/docs/sdd.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+
+### 3.5 Algorithms
+
+`Svc::PassiveRateGroup` has no significant algorithms.
+
+## 4. Module Checklists
+
+Document | Link
+-------- | ----
+Design Checklist | [Link](Checklist_Design.xlsx)
```
Comment (ThibFrgsGmz) [inline Svc/PassiveRateGroup/docs/sdd.md:74]: All links are broken, isn'it ?

**Review by timcanham — APPROVED** [review]

File: Svc/PassiveRateGroup/docs/sdd.md (hunk 10 above)
Comment (LeStarch) [reply Svc/PassiveRateGroup/docs/sdd.md]: Fixed links.

**Pull request merged**